File: 0485-max-consecutive-ones/0485-max-consecutive-ones.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {

        int curr = 0, res = 0;

        for(int i=0; i<nums.length; i++){
            if(nums[i] == 1){
                curr += 1;
            }else{
                curr = 0;
            }
            res = Math.max(res, curr);
        }

        return res;
    }
}

File: 0485-max-consecutive-ones/0485-max-consecutive-ones.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        
        res = ones = 0

        for i in range(len(nums)):
            if nums[i] == 0:
                res = max(res, ones)
                ones = 0
            else:
                ones += 1
        
        return max(res, ones)